### DISCUSSION TITLE
Should we switch to an 88 line length for black?

### DISCUSSION THREAD

**Discussion by PurpleMyst** [PurpleMyst/sansio-lsp-client#26] [Ideas]
This is black's default and 79 is arbitrary anyways.

**Comment by PurpleMyst** [comment]
I've created a [black88 branch](https://github.com/PurpleMyst/sansio-lsp-client/tree/black88) to show off the change.
I think this improves readability in the places that it touches.

**Reply by Akuli** [reply]
I like it. I made a PR from that branch.
